libs/sdk-core/src/models.rs

```diff
@@ -751,11 +762,6 @@ pub struct OpeningFeeParams {
 }
 
 impl OpeningFeeParams {
-    /// Simple validation: checks if `valid_until` is a valid date
-    pub(crate) fn validate(&self) -> Result<()> {
-        self.valid_until_date().map(|_| ())
```

Comment:
Overall looks good.

But is it a good idea to get rid of ofp validation? The result of the new `open_channel_fee` service method will be user input to the other service methods that need custom fees, so we might want to validate that user-inputted fees?

Or should we instead just rely on the LSP? If the LSP accepts the fees, then that's good enough?

Reply:
We need to validate but I only removed it because I saw it is not in use except for the tests.
I also saw that the validation exists here: https://github.com/breez/breez-sdk/blob/main/libs/sdk-core/src/models.rs#L858

Reply:
Good catch